src/main/java/net/minecraftforge/fmp/ForgeMultipart.java

```diff
@@ -0,0 +1,58 @@
+package net.minecraftforge.fmp;
+
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.Mod.EventHandler;
+import net.minecraftforge.fml.common.SidedProxy;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLPostInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+import net.minecraftforge.fmp.block.BlockMultipartContainer;
+import net.minecraftforge.fmp.block.TileCoverable;
+import net.minecraftforge.fmp.block.TileMultipartContainer;
+import net.minecraftforge.fmp.network.MultipartNetworkHandler;
+import net.minecraftforge.fmp.util.MCMPEventHandler;
+
+@Mod(modid = ForgeMultipart.MODID, name = ForgeMultipart.NAME, version = ForgeMultipart.VERSION, acceptedMinecraftVersions = "[1.9,1.9.4]")
+public class ForgeMultipart
+{
+    
+    public static final String MODID = "forgemultipart", NAME = "Forge Multipart", VERSION = "%VERSION%";
+
+    @SidedProxy(serverSide = "net.minecraftforge.fmp.FMPCommonProxy", clientSide = "net.minecraftforge.fmp.client.FMPClientProxy")
+    public static FMPCommonProxy proxy;
+
+    public static BlockMultipartContainer multipart;
+
+    @EventHandler
+    public void preInit(FMLPreInitializationEvent event)
+    {
+        // Register the multipart container Block and TileEntity
+        GameRegistry.register(multipart = new BlockMultipartContainer(), new ResourceLocation(ForgeMultipart.MODID, "multipart"));
+        GameRegistry.registerTileEntityWithAlternatives(TileMultipartContainer.Ticking.class, "mcmultipart:multipart.ticking",
+                "mcmultipart:multipart");
+        GameRegistry.registerTileEntity(TileMultipartContainer.class, "mcmultipart:multipart.nonticking");
+        // Register the default coverable tile for use with blocks that want to host covers, but don't require a TE
+        GameRegistry.registerTileEntity(TileCoverable.class, "mcmultipart:coverable");
+
+        proxy.preInit();
+
+        MinecraftForge.EVENT_BUS.register(new MCMPEventHandler());
```

Comment:
Talked to @cpw about it. It is now an FML mod, just like FML, Forge and MCP. Version is the same as Forge's because I don't need anything fancy for it. We should be able to merge it into Forge's mod class, but it'd only make it more confusing and, atm, I'm trying to keep FMP as a separate "entity" in the system as possible, in case anything happens and it needs to be modified.